# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h
# Commit: bb3495397f18
# Full Hash: bb3495397f18726da8eb3eef7866400c3a495127
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2025-01-24 21:02:20
# Regressor Bug: 1942245
# File Overlap Count: 2
# Description:
#   Bug 1942245 - Remove DataMutex from mInputImageMap. r=aosmond
#   
#   Its use is single threaded.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D234859
# ==============================================================================

diff -r e673cfb778ea -r bb3495397f18 dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h
--- a/dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h	Fri Jan 24 11:43:22 2025 +0000
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h	Fri Jan 24 11:43:23 2025 +0000
@@ -259,7 +259,7 @@
     int64_t timestamp_us;
   };
   // Map rtp time -> input image data
-  DataMutex<std::map<uint64_t, InputImageData>> mInputImageMap;
+  std::map<uint64_t, InputImageData> mInputImageMap;
 
   MediaEventProducer<uint64_t> mInitPluginEvent;
   MediaEventProducer<uint64_t> mReleasePluginEvent;
